docs/docs/api.md

```diff
@@ -822,6 +826,9 @@ $ curl https://manifold.markets/api/v0/market/{marketId}/sell -X POST \
     --data-raw '{"outcome": "YES", "shares": 10}'
 ```
 
+Response type: A `Bet`, with an additional `"status": "success"` field, for backwards
```

Comment:
Hm, instead of mentioning "In previous versions it simply returned `{"status": "success"}`.", here, could you add the message to the changelog section? Thanks!

Reply:
Ok, no prob! I didn't spot that there was a changlelog section